src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -962,7 +962,22 @@ function getWarningMessage() {
  * @param textAreaId - Id of text area for which char are to be counted
  * @param charCountId - Id of Label to display length of text area
  */
-function updateTextQuestionCharCount(textAreaId, charCountId) {
-    var cs = $('#' + textAreaId).val().length;
-    $('#' + charCountId).text(cs);
+function updateTextQuestionCharCount(textAreaId, charCountId, recommendedLength) {
```

Comment:
Do update the names of the method and variables to reflect that words are being counted rather than characters